packages/@uppy/companion/src/server/controllers/list.js

```diff
@@ -2,13 +2,10 @@ const { respondWithError } = require('../provider/error')
 
 async function list({ query, params, companion }, res, next) {
   const { providerUserSession } = companion
-  const { accessToken } = providerUserSession
 
   try {
-    // todo remove backward compat `token` param from all provider methods (because it can be found in providerUserSession)
```

Comment:
I suppose the last two major versions of companion/uppy don't need this anymore? Making it okay to remove?

Reply:
yes. tokens created before [uppy%404.16.0](https://github.com/transloadit/uppy/releases/tag/uppy%404.16.0) 06d9a7c689e5123d41b38191074eb8bbd4ff5325 will become invalid and users who have these old toknes will have to re-authenticate.